bebraven/platform
# Editor slider

**Task**: https://app.asana.com/0/1170776727341290/1174986069483581

**Test**
- 2 new sections, 2 slider questions, change min/max/step
- Click between two, see the slider editing inputs update
<img width="1531" alt="Screen Shot 2020-05-08 at 2 32 01 PM" src="https://user-images.githubusercontent.com/62911934/81451246-e7b7cd00-9138-11ea-8a52-0053ae0d5d7c.png">
<img width="1536" alt="Screen Shot 2020-05-08 at 2 31 55 PM" src="https://user-images.githubusercontent.com/62911934/81451244-e71f3680-9138-11ea-93de-a1a4c8ccd162.png">

- Publish and make sure things behave sanely with our HTML
<img width="1097" alt="Screen Shot 2020-05-08 at 2 29 15 PM" src="https://user-images.githubusercontent.com/62911934/81451267-f7cfac80-9138-11ea-9b07-dd56373044d7.png">

- This is the HTML I found from an existing module `new.html` to test with: https://pastebin.com/8WMy6aGY

- This is the default HTML inserted when you put in a slider question:
<img width="1115" alt="Screen Shot 2020-05-08 at 2 26 48 PM" src="https://user-images.githubusercontent.com/62911934/81451312-0e760380-9139-11ea-82ef-04f77196f629.png">

**Details**

- Commit 1: Refactor out some of the constants used for slider questions (min/max/step). 
- Commit 2: Fix the slider editing input so they update when you click between sliders
- Note: we're not giving designers the option to change `data-bz-range-clg` and `data-bz-range-flr` in the UI right now, but we support it for existing HTML.
